### ISSUE TITLE
Does BDash support MySQL Cleartext plugin?

### ISSUE THREAD

**Issue by davidstosik** [bdash-app/bdash#215]
Hello!

I'd like to connect to an AWS Aurora MySQL database, but doing so requires using MySQL's cleartext plugin.

[Client-Side Cleartext Pluggable Authentication](https://dev.mysql.com/doc/refman/8.0/en/cleartext-pluggable-authentication.html)
[Connecting to your DB instance using IAM authentication from the command line: AWS CLI and mysql client](https://docs.aws.amazon.com/us_en/AmazonRDS/latest/UserGuide/UsingWithRDS.IAMDBAuth.Connecting.AWSCLI.html)

Is there a way to do so with BDash at the moment?

Thank you for your answer!

**Comment by hokaccha** [comment]
No, it is not supported now. Maybe it can be implemented in the following way.
https://github.com/sidorares/node-mysql2/issues/438#issuecomment-255343793
